extension/landing-zone/README.md

```diff
@@ -0,0 +1,9 @@
+# Add a VSI to a landing zone VPC
+
+This logic creates a VSI to an existing landing zone VPC.
+
+## Infrastructure
+
+This code creates and configures the following infrastructure:
+- A new public SSH key, if one is not passed in.
```

Comment:
It doesn't create a new public SSH key. It supports adding a key, or using an existing one